src/main/java/teammates/ui/pagedata/InstructorFeedbackEditPageData.java

```diff
@@ -33,6 +33,7 @@
     private FeedbackQuestionEditForm newQnForm;
     private FeedbackSessionPreviewForm previewForm;
     private String statusForAjax;
+    private String isToBeLoadedInEditMode;
```

Comment:
I think this can also be a boolean? Also, rename to `shouldLoadInEditMode` as it's more concise :)